src/System.Drawing.Design.Primitives/ref/System.Drawing.Design.Primitives.csproj

```diff
@@ -8,6 +8,9 @@
     <Compile Include="System.Drawing.Design.Primitives.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="System.ComponentModel.Primitives" />
```

Comment:
No simple-name references in these projects.  That creates a race condition in the build.  Please use project references.